db/index.js

```diff
@@ -0,0 +1,11 @@
+const { Pool } = require('pg');
+const {
+  user,
+  host,
+  database,
+  password,
+  port,
+} = require('../secrets/db_configuration');
```

Comment:
Not sure if this will work on Heroku: since the secrets directory isn't under source control

Reply:
Is there a way to use .env variables instead?